assets/js/blocks/cart-checkout/cart/attributes.js

```diff
@@ -21,6 +21,10 @@ const blockAttributes = {
 		type: 'boolean',
 		default: getSetting( 'hasDarkEditorStyleSupport', false ),
 	},
+	showRateAfterTaxName: {
+		type: 'boolean',
+		default: getSetting( 'displayCartPricesIncludingTax', false ),
```

Comment:
Should we default this to true? Why is it based on `displayCartPricesIncludingTax`?

Reply:
Good spot! This was a mistake. I think we can just default straight to a boolean value as there's nothing on the back end, besides the inspector control that can decide this. Do you think it should default to `true`? I think defaulting to false would be better, that way it's an opt-in feature rather than opt-out.

Reply:
Thought as much!

I think true, because its something that can be disabled should someone have customised the names, but I think it's more likely that the % is beneficial. I think we should default it to on so the default experience matches the designs.

Sometimes itemised taxes look weird without this, e.g. on my test store I had "Taxes" with a sub-heading below of "Tax". With this true, it was "Tax 20%" which is more useful :)